lib/package.py

```diff
@@ -144,6 +144,8 @@ def _load(self):
                     source[source_name]['ref_to_fetch'] = None
                 if 'archive_src_dir' not in source.values()[0]:
                     source[source_name]['archive_src_dir'] = None
+                if 'shallow' not in source.values()[0]:
+                    source[source_name]['shallow'] = False
```

Comment:
I think "shallow" should be an option to the script, not to a specific package, just like "http_proxy". It is not required by any specific package, so I see no reason to include it in yaml files.